lib/config/aws.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	"github.com/apex/log"
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+func (this *config) GetAws() *aws.Config {
+	if this.Aws == nil {
+		log.Debug("AWS Config not defined. Use default config.")
+		this.Aws = &aws.Config{Region: aws.String("us-east-1")}
```

Comment:
Should we hard code region? I have always loaded it from an envar just to keep it open for whatever region you might need to access. Perhaps what we need is to create a lookup chain like the AWS SDKs use??